Assets/Scripts/Localization/LocalizationDownloader.cs

```diff
@@ -226,9 +226,9 @@ private static void OnDownloadLocalizationComplete(Action onLocalizationDownload
                 // called ProjectPorcupineLocalization-*branch name*. Now we need to move all files from that directory
                 // to Application.streamingAssetsPath/Localization.
                 FileInfo[] fileInfo = localizationFolderInfo.GetFiles();
-                if (fileInfo.Length > 0)
+                if (fileInfo.Length > 2)
```

Comment:
&& fileInfo.All(fi => fi.Name.StartWith("en_US.lang")

(this may have issue as I just wrote it in github; but something like that may do the trick)